tests/e2e/specs/out-of-game/lobby.spec.js

```diff
@@ -94,6 +94,32 @@ describe('Lobby - Page Content (Ranked)', () => {
     checkRanked(true);
     cy.get('[data-cy=ready-button-sword-cross-icon]').should('exist');
   });
+
+  it('Toggles ranked/casual switch with "Enter" key', () => {
+    // Focus the ranked/casual switch
+    cy.get('[data-cy=edit-game-ranked-switch]').focus();
```

Comment:
Good idea, though focusing the switch is a little more nuanced, unfortunately, due to the fact that cypress can only focus the input element _inside_ the switch, and that the data-cy attribute gets put on a containing div, not the `input` itself. Try this
```suggestion
    cy.get('[data-cy=edit-game-ranked-switch] input').focus();
```

to select the descendant `<input>` element of the div with the chosen selector.

FOr reference, here is the [cypress cloud run showing the test failure](https://cloud.cypress.io/projects/i8bxr8/runs/2632/overview/38e29299-ad4c-405c-9745-67dbff1fd536/replay?roarHideRunsWithDiffGroupsAndTags=1&runStatus=FAILED&runStatus=RUNNING&runStatus=CANCELLED&runStatus=ERRORED&runStatus=OVERLIMIT&runStatus=PASSED&ts=1739900551207.3&att=1).

You can see the test runs in the "Checks" tab of the PR, and then at the bottom of the logs for a given test run you can find the link to its cypress cloud run, which will let you replay all the tests interactively:
![image](https://github.com/user-attachments/assets/ea4c89ff-000e-45cd-8c61-a2f6bc1fadeb)